includes/class-wp-job-manager-cache-helper.php

```diff
@@ -128,6 +129,105 @@ public static function clear_expired_transients() {
+	 * need to be removed, and remove the
+	 *
+	 * @since 1.6.2
+	 *
+	 * @param string  $new_status New post status.
+	 * @param string  $old_status Old post status.
+	 * @param WP_Post $post       Post object.
+	 */
+	public static function maybe_clear_count_transients( $new_status, $old_status, $post ) {
+		global $wpdb;
+
+		/**
+		 * Get supported post types for count caching
+		 *
+		 * @since 1.26.2
+		 *
+		 * @param string  $new_status New post status.
+		 * @param string  $old_status Old post status.
+		 * @param WP_Post $post       Post object.
+		 */
+		$post_types = apply_filters( 'job_manager_count_cache_supported_post_types', array( 'job_listing' ), $new_status, $old_status, $post );
+
+		// Only proceed when statuses do not match, and post type is supported post type
+		if ( $new_status === $old_status || ! in_array( $post->post_type, $post_types ) ) {
+			return;
+		}
+
+		/**
+		 * Get supported post statuses for count caching
+		 *
+		 * @since 1.26.2
+		 *
+		 * @param string  $new_status New post status.
+		 * @param string  $old_status Old post status.
+		 * @param WP_Post $post       Post object.
+		 */
+		$valid_statuses = apply_filters( 'job_manager_count_cache_supported_statuses', array( 'pending' ), $new_status, $old_status, $post );
+
+		$rlike = array();
+		// New status transient option name
+		if( in_array( $new_status, $valid_statuses ) ){
+			$rlike[] = "^_transient_jm_{$new_status}_{$post->post_type}_count_user_";
+		}
+		// Old status transient option name
+		if( in_array( $old_status, $valid_statuses ) ){
+			$rlike[] = "^_transient_jm_{$old_status}_{$post->post_type}_count_user_";
+		}
+
+		$sql        = $wpdb->prepare( "SELECT option_name FROM $wpdb->options WHERE option_name RLIKE '%s'", implode('|', $rlike ) );
+		$transients = $wpdb->get_col( $sql );
+
+		// For each transient...
+		foreach ( $transients as $transient ) {
+			// Strip away the WordPress prefix in order to arrive at the transient key.
+			$key = str_replace( '_transient_', '', $transient );
+			// Now that we have the key, use WordPress core to the delete the transient.
+			delete_transient( $key );
+		}
+
+		// Sometimes transients are not in the DB, so we have to do this too:
+		wp_cache_flush();
+	}
+
+	/**
+	 * Get Listings Count from Cache
+	 *
+	 *
+	 * @since 1.26.2
+	 *
+	 * @param string $post_type
+	 * @param string $status
+	 * @param bool   $force Force update cache
+	 *
+	 * @return int
+	 */
+	public static function get_listings_count( $post_type = 'job_listing', $status = 'pending', $force = FALSE ) {
+
+		// Get user based cache transient
+		$user_id   = get_current_user_id();
+		$transient = "jm_{$status}_{$post_type}_count_user_{$user_id}";
+
+		// Set listings_count value from cache if exists, otherwise set to 0 as default
+		$status_count = ( $cached_count = get_transient( $transient ) ) ? $cached_count : 0;
+
+		// $cached_count will be false if transient does not exist
+		if ( $cached_count === FALSE || $force ) {
```

Comment:
Small formatting fix: Yoda; lowercase `false` (same with function signature's default for `$force`).